src/arkouda_server.chpl

```diff
@@ -81,13 +106,23 @@ proc main() {
     }
 
     socket.bind("tcp://*:%t".format(ServerPort));
-
-    const boundary = "**************************************************************************" +
-                   "**************************";
-
-    asLogger.info(getModuleName(), getRoutineName(), getLineNumber(), boundary);
-    asLogger.info(getModuleName(), getRoutineName(), getLineNumber(), serverMessage);
```

Comment:
@hokiegeek2 I think with this change the above "server listening on..." message is no longer printed. (Noticed while trying to do some manual server runs and not seeing the host/port printed anymore)

Reply:
@ronawho not sure what you mean; this is from a fresh build of the main branch:

![image](https://user-images.githubusercontent.com/10785153/112331790-9e5c6780-8c8f-11eb-8494-e6d34db7d143.png)

The "server listening on" message is there.

Reply:
Huh, ok. So I see what you see on my local machine, but on a new machine I'm running on I just see:

```
$ ./arkouda_server -nl 1
2021-03-24:15:10:08 [arkouda_server] main Line 64 INFO [Chapel] arkouda server version = 2020.07.07
2021-03-24:15:10:08 [arkouda_server] main Line 66 INFO [Chapel] memory tracking = true
2021-03-24:15:10:08 [arkouda_server] main Line 69 INFO [Chapel] initialized the .arkouda directory /home/eronaghan/arkouda-sort-subtimers/.arkouda
2021-03-24:15:10:08 [arkouda_server] main Line 73 INFO [Chapel] getMemLimit() 1947559340851
2021-03-24:15:10:08 [arkouda_server] main Line 75 INFO [Chapel] bytes of memoryUsed() = 39472
```

And then I don't get anything else until I connect a client to it. It looks like the `socket.bind` call is blocking until a client connects on this system. Ok, so not a new problem and the behavior didn't change with this PR. Sorry for the noise.

Reply:
Actually, some of our internal users are seeing the same thing with recent builds of arkouda, whereas a build from 2/26 works fine on the same system. It's a pretty big problem on a multi-node system, because the user doesn't know the hostname to connect to. If the problem is a blocking `socket.bind`, can we just print out the connect string before calling `bind`?

Reply:
@reuster986 Okay, I will put a logging message w/ connect info back in right before bind and will do so in #715

Reply:
@reuster986 @ronawho put print message including connect string before socket.bind. @ronawho please pull down my latest [PR](https://github.com/mhmerrill/arkouda/pull/716) and see if the print message is seen on startup. Sorry, I cannot replicate this on my mac, so I did not see the issue you guys are reporting.

Reply:
@reuster986 @ronawho Also added stdout.flush() at @mhmerrill suggestion. I have that in the logging framework I wrote but not in the printServerSplashMessage. From what I understand from my discussion with Mike, that might explain what y'all are seeing on the Crays